AU2/plugins/custom_plugins/ScoringPlugin.py

```diff
@@ -131,6 +214,160 @@ def answer_set_bonuses(self, html_response: Dict[str, Any]) -> List[HTMLComponen
+            ))
+        table_str = stats_table_template(columns).format(ROWS="".join(rows))
+
+        # kill tree visualiser
+        killtree_embed = ""
+        killtree_link = ""
+        if generate_killtree:
+            # skip if we don't have pyvis or lxml installed
+            try:
+                from pyvis.network import Network
+                import lxml.html
+
+                # track competency and wantedness for edge colouring
+                competency_manager = CompetencyManager(get_game_start())
+                wanted_manager = WantedManager()
+
+                net = Network(directed=True, cdn_resources="in_line", height="calc(100vh - 90px)", select_menu=True)
+                NODE_SHAPE = "dot"
+                added_nodes = set()
+                for e in events:
+                    # construct kill tree network
+                    for (killer, victim) in e.kills:
+                        competency_manager.add_event(e)
+                        wanted_manager.add_event(e)
+                        killer_model = ASSASSINS_DATABASE.get(killer)
+                        victim_model = ASSASSINS_DATABASE.get(victim)
+                        killer_searchable = f"{killer_model.all_pseudonyms()} ({killer_model.real_name})"
+                        victim_searchable = f"{victim_model.all_pseudonyms()} ({victim_model.real_name})"
+                        if killer not in added_nodes:
+                            net.add_node(
+                                killer_searchable,
+                                label=killer_model.real_name + (" (Police)" if killer_model.is_police else ""),
+                                shape=NODE_SHAPE,
+                                color=get_color(killer_model.get_pseudonym(0), is_police=killer_model.is_police),
+                                title=killer_searchable,
+                                value=1+score_manager.get_conkers(killer_model)
+                            )
+                            added_nodes.add(killer)
+                        if victim not in added_nodes:
+                            net.add_node(
+                                victim_searchable,
+                                label=victim_model.real_name + (" (Police)" if victim_model.is_police else ""),
+                                shape=NODE_SHAPE,
+                                color=get_color(victim_model.get_pseudonym(0), is_police=victim_model.is_police),
+                                title=victim_searchable,
+                                value=1+score_manager.get_conkers(victim_model)
+                            )
+                            added_nodes.add(victim)
+                        headline = render_headline_and_reports(
+                            e,
+                            death_manager=DeathManager(),
+                            competency_manager=competency_manager,
+                            wanted_manager=wanted_manager
+                        )[0]

```

Comment:
alternatively, `*_` if you get more than one item out of this function. make it obvious at the front you're getting more than one thing out of this function and throwing all but one thing away

```suggestion
                        headline, _ = render_headline_and_reports(
                            e,
                            death_manager=DeathManager(),
                            competency_manager=competency_manager,
                            wanted_manager=wanted_manager
                        )
```